wp-content/themes/csisjti/archive-resource-library.php

```diff
@@ -19,7 +19,7 @@
 	<div class="resource-library__top-bar"></div>
 	<div class="resource-library__content">
 		<div class="resource-library__secondary">
-			<div class="resource-library__sticky">
+			<div class="resource-library__side-panel">
```

Comment:
I think we can actually remove the `source-library__side-panel` div completely. It only existed to add in the sticky property, so since we're removing it, it doesn't need to be here.